lib/inventory.rb

```diff
@@ -68,5 +69,18 @@ def restore(path)
     FileUtils.copy(backup_path, path)
     true
   end
+
+  def group(group_name)
+    result = {}
+    each do |name, value|
+      if name =~ /^#{group_name}\.(.+)$/
```

Comment:
__[rubocop]__
* Style/Next: Use `next` to skip iteration. [:ref](https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)

<!-- Comment by [SideCI](https://sideci.com) -->